api/Jobs.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: drazen.popov
+ * Date: 9/22/2015
+ * Time: 10:28 AM
+ */
+include_once'autoload/boot.php';
+class Jobs {
+
+    public $arg;
+    public $arg1;
+    public $action;
+
+    function __construct($action , $arg ,$arg1 )
+    {
+
+        $this->arg = $arg;
+        $this->arg1 = $arg1;
+        $this->action = $action;
+    }
+
+    function read() {
+
+        try {
+
+            $conn = new DbConn();
```

Comment:
It is unnecessary to have DB access in all of the CRUD methods. Better way would be to make DB connection out of them and store connection in $link property of the class, so you can re-use the same connection everywhere.